zend_abstract_interface/interceptor/php8/resolver_pre-8_2.c

```diff
@@ -35,9 +59,6 @@ static zend_op_array *zai_interceptor_compile_file(zend_file_handle *file_handle
 
     zend_op_array *op_array = prev_compile_file(file_handle, type);
 
-    classpos = zend_hash_iterator_pos(class_iter, CG(class_table));
-    funcpos = zend_hash_iterator_pos(func_iter, CG(function_table));
-
     zai_interceptor_add_new_entries(classpos, funcpos);
```

Comment:
**question:** we still need this in the PHP7 resolver. I have to admit, that I do not fully understand this detail. Why do we not need to call this anymore?